core/controllers/admin.py

```diff
@@ -288,7 +288,12 @@ def post(self):
                 result = {
                     'version': version
                 }
-            elif action == 'update_feature_flag_rules':
+            # The handler schema defines the possible values of 'action'.
```

Comment:
@IamEzio How about doing the same thing here as you did in the other file, i.e. use "else" and assert and a comment? That seems foolproof IMO.

Ditto elsewhere.

Reply:
Agreed and I think that approach should be documented and used everywhere.

Reply:
Done